packit_service/worker/checker/distgit.py

```diff
@@ -94,22 +94,24 @@ def pre_check(self) -> bool:
                     return False
         elif self.data.event_type in (PullRequestCommentPagureEvent.__name__,):
             commenter = self.data.actor
+            tagging = "koji-tag" in self.data.event_dict.get("comment", "")
```

Comment:
Can't I do `/packit koji-tag koji-build`?

Reply:
I suppose you can, but you will get the same result as with `/packit koji-tag`. But I get your point, `/packit koji-build koji-tag` would be problematic. Let me fix that.

Reply:
thanks :+1: